rollup-tests.config.mjs

```diff
@@ -39,6 +39,7 @@ export default {
     }),
     nodeResolve({
       browser: true,
+      extensions: ['.js', '.ts', '.tsx'],
```

Comment:
This allows Rollup to match imports like `import { Foo } from './some-module'` to match any of `some-module.{js, ts, tsx}`.